supply-chain/config.toml

```diff
@@ -28,7 +28,7 @@ criteria = "safe-to-deploy"
 
 [[exemptions.adler]]
 version = "1.0.2"
-criteria = "safe-to-deploy"
+criteria = "safe-to-run"
```

Comment:
Can you say more about this exemption update? As far as I understood it, we decided we wouldn't add any more exemptions past the initial set that we had when we adopted cargo-vet, so this might need a true audit rather than an expanded exemption. (But I don't claim to fully understand our policies and cargo-vet subtleties here, so others please feel free to chime in too...)

Reply:
Ah, actually, reading more, it seems that `safe-to-run` is a *subset* of `safe-to-deploy` (a weaker claim), so this is a downgrade and should be automatically fine; does that match your understanding? Was this change due to some indication that we didn't need the wider claim anymore?

Reply:
I didn't think about it too hard because this is something `cargo vet` did automatically, and I figured it knew what it was doing. But I believe this is weakening the exemption. If I understand correctly, `safe-to-deploy` implies `safe-to-run`, but not the other way around, so with this change we're no longer saying we trust these exempted crates as much as we did before.

Reply:
Jinx :laughing:

All the changes to `config.toml` are because Miette had a bunch of dependencies while codespan-reporting only depends on a couple of crates, which I guess we were already using elsewhere. `termcolor` is used by `clap`, `cranelift-tools`, and others, while `unicode-width` is used by `clap`, `wast`, and others.